src/lib/utils/getCommonPinningStyles.ts

```diff
@@ -0,0 +1,18 @@
+import { Column } from "@tanstack/react-table";
+import { CSSProperties } from "react";
+
+// These are the important styles to make sticky column pinning work!
+const getCommonPinningStyles = <TData>(column: Column<TData>): CSSProperties => {
+  const isPinned = column.getIsPinned();
+
+  return {
+    left: isPinned === "left" ? `${column.getStart("left")}px` : undefined,
+    right: isPinned === "right" ? `${column.getAfter("right")}px` : undefined,
+    opacity: isPinned ? 1 : 1,
```

Comment:
@skuengneo You always apply 1 as opacity, are you sure this is correct?

Reply:
Removed it. The tanstack example had 0.95 for sticky but that looked awful. Maybe we could add style options for sticky columns in the future if anyone needs it